peps/pep-0768.rst

```diff
@@ -310,6 +330,10 @@ processes that can already write to arbitrary memory within your process and
 execute arbitrary code on the machine (in order to create the file containing
 the Python code to be executed).
```

Comment:
This file is no longer created.

Reply:
I suppose you wrote this before checking out the other PR but just to clarify: in the new model the debugger tool must write a file to be executed